tests/test.sh

```diff
@@ -13,15 +16,20 @@ test() {
   name=$1; shift
   expect=$1; shift
   cd $name
```

Comment:
seeing as we're inside a function here, we should be working with `pushd` / `popd`